<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CStudent</class>
 <widget class="QWidget" name="CStudent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_ConfirmExit">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>590</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确认退课</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ConfirmSelect">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确认选课</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Start">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开始查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>已选课程:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Welcome">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>160</y>
     <width>161</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>同学欢迎进入</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_Sum">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>460</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>已修学分：</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Select">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>全部课程</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>本专业课程</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>选修课程</string>
    </property>
   </item>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>611</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>611</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>60</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../img/img.qrc">:/StudentBackground.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>-20</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../img/img.qrc">:/MangerPix.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonChangePassword">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>630</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>修改密码</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../img/img.qrc"/>
 </resources>
 <connections/>
</ui>
